app/models/aleph_item.rb

```diff
@@ -54,7 +54,14 @@ def process_item(item, oclc, scan)
       available?: available?(item),
       label: label(item),
       description: description(item),
-      buttons: ButtonMaker.new(item, oclc, scan).all_buttons }
+      buttons: all_buttons(item, oclc, scan) }
+  end
+
+  def all_buttons(item, oclc, scan)
+    [ButtonContact.new(item, oclc, scan).html_button,
+     ButtonHoldRecall.new(item, oclc, scan).html_button,
```

Comment:
There is ~absolutely no~ circumstance where we could end up with both a hold button and a recall button, right? Like, this doesn't need to be something like
```
ButtonHoldRecall.new(item, oclc, scan, mode='hold').html_button,
ButtonHoldRecall.new(item, oclc, scan, mode='recall').html_button,
```
?

(I will duly glare at myself for having used modes here.)

Reply:
Ha. I was thinking of adding in tests for this explicitly. In the tests whenever we check for one check the other is the opposite. I'll add that to the todos (I'm confident this is a non-issue but the tests would be super easy to add so why not)

Reply:
Will do... plus the extra tests I noted.

Reply:
Those extra tests sound great.